netplan/cli/commands/status.py

```diff
@@ -157,9 +156,10 @@ def query_nm_ssid(self, con_name: str) -> str:
 
     def query_networkctl(self, ifname: str) -> str:
         output: str = None
+        env = dict(LC_ALL='C', PATH=os.environ.get('PATH', utils.FALLBACK_PATH))
         try:
             output = subprocess.check_output(['networkctl', 'status', ifname],
```

Comment:
Not relevant to this PR so feel free to ignore.
It's very uncommon but network interfaces having a name that starts with `-` is a thing.
I would recommend doing:
```python
['networkctl', 'status', '--', ifname]
```
just to avoid any problem.

```
$ ip link add -lo type dummy
$ networkctl status -lo
networkctl: invalid option -- 'o
$ networkctl status -- -lo
● 37: -lo                                                               
                     Link File: /usr/lib/systemd/network/99-default.link
[...]
```